src/openms/source/QC/DBSuitability.cpp

```diff
@@ -597,6 +616,78 @@ namespace OpenMS
     return -(db_slope) / (deNovo_slope);
   }
 
+  UInt DBSuitability::numberOfUniqueProteins_(const std::vector<PeptideIdentification>& peps, UInt number_of_hits) const
+  {
+    set<String> proteins;
+
+    for (const auto& pep : peps)
+    {
+      const vector<PeptideHit>& hits = pep.getHits();
+      for (UInt i = 0; i < number_of_hits; ++i)
```

Comment:
possible reason why the number of proteins does not change much during subsamping: you use ALL hits.
Better use the top hit only?

Reply:
Ahh.. Yes, that could be the reason. I'll try that.